<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceSlider</class>
 <widget class="QWidget" name="SliceSlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>49</width>
    <height>312</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>50</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelTotalLayers">
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>145</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="curNumOfLayersSlider">
     <property name="maximum">
      <number>145</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEditCurNumOfLayers">
     <property name="minimumSize">
      <size>
       <width>38</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
